spec/directives/supports.hrx

```diff
@@ -0,0 +1,24 @@
+<===> error/syntax/input.scss
```

Comment:
Since this isn't the only syntax error we'll ever want to test, it should drill deeper than this... maybe `error/syntax/trailing_or`?

Reply:
Done